frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -326,6 +326,23 @@ export function Table({
     [serverSidePagination, clientSidePagination]
   );
 
+  registerAction(
+    'selectRow',
+    async function (id) {
+      const item = currentState.components[component.name]['currentData'].filter((item) => item.id == id);
+      const row = page.find((item, index) => item.original.id == id);
+      const rowId = row.id;
+      const original = row.original;
```

Comment:
Can you remove the `original` variable if it is not needed? And we can use `row.id` directly in the object instead of creating a new variable